integration-cli/docker_cli_run_test.go

```diff
@@ -3186,3 +3186,24 @@ func (s *DockerSuite) TestRunUnshareProc(c *check.C) {
 		c.Fatalf("unshare should have failed with permission denied, got: %s, %v", out, err)
 	}
 }
+
+func (s *DockerSuite) TestRunPublishPort(c *check.C) {
+	testRequires(c, NativeExecDriver)
+	name := "busybox:port"
+	dockerFile := fmt.Sprintf("FROM busybox\nEXPOSE 8080")
+	if _, err := buildImage(name, dockerFile, false); err != nil {
```

Comment:
Instead of building an image, can you just do `docker run --expose 8080 busybox`?
Also, please use `c.Assert` to check errors.